### PR TITLE
docs: Document support for service-wide layers

### PR DESCRIPTION
**Pull request by rmorey**
This was added in #5582 but is not present in the documentation.

Added one example, taken directly from the linked issue.

### CODE REVIEW

File: docs/providers/aws/guide/layers.md [hunk 1]
```diff
@@ -176,3 +176,23 @@ functions:
...
+
+You can also configure layers at the service level. EG:
+
+```yml
+# serverless.yml
+service: aws-nodejs
+
+provider:
+  name: aws
+  runtime: nodejs8.10
```
Comment (pgrzesik) [inline docs/providers/aws/guide/layers.md:188]: Let's not use an outdated runtime in the example

**Review by pgrzesik — CHANGES_REQUESTED** [review]
good catch @rmorey :bow: I just have one minor comment and we're good to go

File: docs/providers/aws/guide/layers.md (hunk 1 above)
Comment (rmorey) [reply docs/providers/aws/guide/layers.md:188]: Good call. Lazy copy-paste on my part

**Review by pgrzesik — APPROVED** [review]
Thank you @rmorey :bow:

**Pull request merged**